client/oslc-client/src/main/java/org/eclipse/lyo/client/OslcClient.java

```diff
@@ -33,577 +29,477 @@
-		return resource.readEntity(clazz);
-	}
-
-	/**
-	 * Gets OSLC resources in parallel from an array of URIs and unwraps their corresponding entities.
-	 */
-	public <T> List<T> getResources(final URI[] links, final Class<T> clazz) {
-		return Arrays.stream(links).parallel().map(uri -> {
-			final Response resource = getResource(uri.toString());
-			return resource.readEntity(clazz);
-		}).collect(Collectors.toList());
-	}
-
-	/**
-	 * Gets OSLC resources in parallel from a collection of URIs and unwraps their corresponding entities.
-	 */
-	public <T> List<T> getResources(final Collection<URI> links, final Class<T> clazz) {
-		return links.parallelStream().map(uri -> {
-			final Response resource = getResource(uri.toString());
-			return resource.readEntity(clazz);
-		}).collect(Collectors.toList());
-	}
-
-	/**
-	 * Gets OSLC resources in parallel from a set of Links and unwraps their corresponding entities.
-	 * <p>
-	 * Method renamed due to type erasure in Java
-	 */
-	public <T> List<T> getResourcesFromLinks(final Collection<Link> links, final Class<T> clazz) {
-		final URI[] uris = links.stream().map(l -> l.getValue()).toArray(URI[]::new);
-		return getResources(uris, clazz);
-	}
-
-	/**
-	 * Gets an OSLC resource using <code>application/rdf+xml</code>. Use
-	 * {@link #getResource(String, String)} to specify the media type or
-	 * {@link #getResource(String, Map)} to add other request headers.
-	 */
-	@SuppressWarnings("unused")
-	public Response getResource(String url) {
-		return getResource(url, null, OSLCConstants.CT_RDF, null, true);
-	}
-
-	/**
-	 * Gets an OSLC resource. Use {@link #getResource(String, Map)} instead to
-	 * add other request headers.
-	 *
-	 * @param url
-	 *            the resource URL
-	 * @param mediaType
-	 *            the requested media type to use in the HTTP Accept request
-	 *            header
-	 */
-	@SuppressWarnings("unused")
-	public Response getResource(String url, final String mediaType) {
-		return getResource(url, null, mediaType, null, true);
-	}
-
-	/**
-	 * Gets an OSLC resource.
-	 *
-	 * @param url
-	 *            the resource URL
-	 * @param requestHeaders
-	 *            the HTTP request headers to use. If the <code>Accept</code>
-	 *            header is not in the map, it defaults to
-	 *            <code>application/rdf+xml</code>. If
-	 *            <code>OSLC-Core-Version</code> is not in the map, it defaults
-	 *            to <code>2.0</code>.
-	 */
-	public Response getResource(String url, Map<String, String> requestHeaders) {
-		return getResource(url, requestHeaders, OSLCConstants.CT_RDF, null, true);
-	}
-
-	public Response getResource (String url, Map<String, String> requestHeaders, String defaultMediaType) {
-	    return getResource(url, requestHeaders, defaultMediaType, null, true);
-	}
-
-    public Response getResource (String url, Map<String, String> requestHeaders, String defaultMediaType,
+    private final String version;
```

Comment:
The whole file has been changed. 
Main reason is due to white-space changes. But other formatting changes are also introduced.
Please revert such changes unless they are intentional.